thoth/package_extract/image.py

```diff
@@ -464,6 +465,21 @@ def construct_rootfs(dir_path: str, rootfs_path: str) -> list:
     return layers
 
 
+def _get_absolute_link(root_path: str, path: str, iter: int) -> str:
+    """Find the absolute link of the given link."""
+    if iter > _MAX_SYMLINKS:
+        return None
+    if os.path.islink(path):
+        next_link = root_path
+        readlink = os.readlink(path)
+        if readlink[0] != '/':
+            next_link += '/usr/bin/'
```

Comment:
Something like `next_link = os.path.join(path, os.path.basename(path))`? It does not need to be always `/usr/bin/`.

Reply:
> Something like `next_link = os.path.join(path, os.path.basename(path))`? It does not need to be always `/usr/bin/`.

@fridex My thought was that if `readlink` is something like `python3` or `python3.7`, we definitely know that it is located at `/usr/bin/` globbing that specific part. And, vice versa, if we get something like `/etc/.../` then we should concatenate it with `root_path` which would be something like `/tmp/.../rootfs/

Reply:
The issue might arise in next iterations, when links will point to, for example, `/etc/alternatives/python3`, and here `python3` will point to `python3.7` which would in fact be `/etc/alternatives/python3.7` instad of `/usr/bin/python3.7`.